5.5/Dockerfile

```diff
@@ -18,14 +18,14 @@ ENV IMAGE_EXPOSE_SERVICES   3306:mysql
 MAINTAINER  Martin Nagy <mnagy@redhat.com>
 EXPOSE 3306
 
-RUN groupadd -g 27 -o -r mysql && \
-    useradd -M -N -g mysql -o -r -d /var/lib/mysql -s /sbin/nologin -c "MySQL Server" -u 27 mysql
-
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
     yum -y --setopt=tsflags=nodocs install hostname mysql55 && \
     yum clean all
 
+# Make sure the mysql user is installed with UID 27, so we're compatible across volumes.
+RUN test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)"
```

Comment:
since the test is there, we're still controlling the uid.  and i tend to agree w/ the principal that we shouldn't be mucking about w/ things the RPM is trying to do.

I forget, what happens if we create the user but do NOT create it w/ uid 27, and then install the RPM?